pages/add-contact.js

```diff
@@ -0,0 +1,11 @@
+import Footer from '../src/components/Footer';
+import MyContactDetailForm from '../src/components/MyContactDetailForm';
+
+export default function addContactPage() {
+	return (
+		<>
+			<MyContactDetailForm />
+			<Footer />
+		</>
+	);
+}
```

Comment:
If you want to integrate a footer to every page you should use a Layout component which defines the layout of your pages.